### DISCUSSION TITLE
Special Purpose Operating System WG / Immutable OSes

### DISCUSSION THREAD

**Discussion by sayanchowdhury** [cncf/maintainer-summit#3] [Maintainer Summit India 2024]
The Special Purpose Operating System Working Group (WG) is behind the effort to cater to special purpose operating systems. The unconference plans to introduce the WG and invite folks to join. It would also be a good space for knowledge sharing and discussing how people use immutable OSes for their workloads or newer initiatives like WASM and others.

**Comment by palnabarun** [comment]
This session is accepted as an unconference. Please take notes in this document: https://docs.google.com/document/d/1C5sEx9VHtlzop0wcyPc4HIKyx_3TG1U_Co0YltWu4J4/edit?usp=sharing
